cob_cartesian_controller/CMakeLists.txt

```diff
@@ -4,16 +4,16 @@ project(cob_cartesian_controller)
 find_package(catkin REQUIRED COMPONENTS cmake_modules cob_srvs geometry_msgs roscpp roslib std_msgs std_srvs tf visualization_msgs)
```

Comment:
We significantly restructured `cob_cartesian_controller` but haven't made it a PR yet...in the new version `TinyXML` will no longer be needed....adding it for now anyways...